source/release/4.5.0/index.md

```diff
@@ -72,69 +76,72 @@ dnf config-manager --set-enabled ovirt-45-upstream-testing
+4) Reboot the host after upgrade
 
 
-To check if provider and OVN was configured successfully on the host, webadmin includes OVN configured flag in General tab for every host, at the same time this value is available via REST API. 
+To check if provider and OVN was configured successfully on the host, webadmin includes OVN configured flag in General tab for every host, at the same time this value is available via REST API. Please note that this value might say "No" until capabilities are refreshed for the host.
 
-Fixing the host's OVN not being confiugred can be done by reinstalling it from engine 4.5 or higher.
- - [BZ 1782056](https://bugzilla.redhat.com/show_bug.cgi?id=1782056) **[RFE] Integration of built-in ipsec feature in RHV/RHHI-V with OVN**
+Fixing the host's OVN not being configured can be done by reinstalling it from engine 4.5 or higher.
+ - [BZ 1782056](https://bugzilla.redhat.com/show_bug.cgi?id=1782056) **[RFE] Integration of built-in ipsec feature in oVirt with OVN**
    The IPSec for OVN feature is available on hosts with configured ovirt-provider-ovn, OVN version 2021 or higher and OvS version 2.15 or higher.
 
 #### oVirt Open vSwitch
 
- - [BZ 1782056](https://bugzilla.redhat.com/show_bug.cgi?id=1782056) **[RFE] Integration of built-in ipsec feature in RHV/RHHI-V with OVN**
+ - [BZ 1782056](https://bugzilla.redhat.com/show_bug.cgi?id=1782056) **[RFE] Integration of built-in ipsec feature in oVirt with OVN**
    The IPSec for OVN feature is available on hosts with configured ovirt-provider-ovn, OVN version 2021 or higher and OvS version 2.15 or higher.
- - [BZ 1940824](https://bugzilla.redhat.com/show_bug.cgi?id=1940824) **[RFE] Upgrade OVN/OVS 2.11 in RHV to OVN/OVS 2.15**
+ - [BZ 1940824](https://bugzilla.redhat.com/show_bug.cgi?id=1940824) **[RFE] Upgrade OVN/OVS 2.11 in oVirt to OVN/OVS 2.15**
    Upgrade from OvS/OVN 2.11 to OVN 2021 and OvS 2.15.
 
 The upgrade is transparent to user as long as few conditions are met:
 
 1) The engine must be upgraded first.
 2) Before any host upgrade the ovirt-provider-ovn security groups must be disabled for all OVN networks that are expected to work between host with OVN/OvS version 2.11.
 3) Upgrade hosts to match the OVN version 2021 or higher and OvS version to 2.15. This step should be done via the UI, in order to reconfigure OVN and refresh certificates.
+4) Reboot the host after upgrade
 
 
-To check if provider and OVN was configured successfully on the host, webadmin includes OVN configured flag in General tab for every host, at the same time this value is available via REST API. 
+To check if provider and OVN was configured successfully on the host, webadmin includes OVN configured flag in General tab for every host, at the same time this value is available via REST API. Please note that this value might say "No" until capabilities are refreshed for the host.
```

Comment:
I'm not sure what this means. I think it might mean this:

To verify that the provider and the OVN were configured successfully, launch the web console and check the "OVN configured" flag on the "General" tab for each host. You can also obtain the value using the REST API. Note that the value might be set to "No" until the host configuration is refreshed.

Reply:
Can you please review doctext at https://bugzilla.redhat.com/show_bug.cgi?id=1940824 ? this is autogenerated from there.

Reply:
Odd -- I can't find the OVN setting in the documentation on [Host configuration](https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/administration_guide/index#Host_General_Settings).

Reply:
@almusil @hbraha @erav can you please check?

Reply:
There is just a field in `General` tab for every host. No configuration or anything. By the refreshed capabilities is meant the `Refresh Capabilites` action that is available for every host. Is that clearer?

Reply:
I would put quotation marks around ` 'OVN configured' flag ` to make it clearer that this is the name of the flag and not some configuration option\facility.

Reply:
> @erav Can you provide a screenshot for the field in the UI? I do not see anything in the docs that seems to be related to the OVN configuration in the Host > "General" settings.

@apinnick Information for this is being added with https://bugzilla.redhat.com/show_bug.cgi?id=2006228. See also PR https://github.com/oVirt/ovirt-site/pull/2779.

Reply:
![image](https://user-images.githubusercontent.com/4718090/161687434-7cf6b82b-8829-4ec7-8512-0f24f234edd0.png)

Reply:
@erav Perfect! Thanks

Reply:
@sandrobonazzola 
I updated the doctext for the bug.

Reply:
Thanks, re-generated the release notes